lib/stripe/dispute.rb

```diff
@@ -10,12 +10,14 @@ class Dispute < APIResource
     custom_method :close, http_verb: :post
 
     def close(params = {}, opts = {})
-      resp, opts = request(:post, close_url, params, opts)
+      resp, opts = request(:post, resource_url + "/close", params, opts)
       initialize_from(resp.data, opts)
     end
 
     def close_url
       resource_url + "/close"
     end
+    extend Gem::Deprecate
+    deprecate :close_url, :none, 2019, 5
```

Comment:
Just realized that these deprecation warnings generate the following message: 

```
NOTE: Stripe:: Dispute#close_url is deprecated with no replacement. It will be removed on or after 2019-05-01.
```

... which doesn't seem accurate regarding the deprecation date. 

Any suggestion what we should put there instread?

Reply:
The date is okay IMO, but there is a replacement for this method! Could you change it to `deprecated :close_url, "Dispute.close", 2019, 5` instead?

Reply:
(Deleted my previous comment, I was confused.)

The date is okay IMO, it just means that the method is liable to disappear at any point after 2019-05-01. @brandur-stripe do you agree?

Reply:
Yeah, exactly what OB said — it's a warning, but without a firm commitment on exactly when.

If you both agree, I'd also be okay just choosing a date ~6 months in the future or so like 2019-11-01. We're very unlikely to remove before then anyway, and this is somewhat better form in that we're allowing users a little more grace time.